incubator/perfsonar-testpoint-demo/perfsonar-testpoint-demo/templates/deployment.yaml

```
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "perfsonar-testpoint-demo.fullname" . }}
+  labels:
+    app: {{ template "perfsonar-testpoint-demo.name" . }}
+    chart: {{ template "perfsonar-testpoint-demo.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+    instanceID: {{ .Values.SLATE.Instance.ID | quote  }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "perfsonar-testpoint-demo.name" . }}
+      chart: {{ template "perfsonar-testpoint-demo.chart" . }}
+      release: {{ .Release.Name }}
+      instance: {{ .Values.Instance | quote }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: {{ template "perfsonar-testpoint-demo.name" . }}
+        chart: {{ template "perfsonar-testpoint-demo.chart" . }}
+        release: {{ .Release.Name }}
+        instance: {{ .Values.Instance | quote }}
+    spec:
+      nodeSelector:
+        # For Node Selection
+        {{ if .Values.NodeSelection.Hostname }}
+        kubernetes.io/hostname: {{ .Values.NodeSelection.Hostname }}
+        {{ end }}
+      hostNetwork: true
+      containers:
+      {{ if .Values.HTTPLogger.Enabled }}
```

```suggestion
      {{ if .Values.HTTPLogger.Enabled }}
```
I'd like to simplify this by removing the HTTPLogger. As far as I can tell, we don't need to serve multiple files here, just the test results, so lets send that to `stdout` so that it will appear in `slate instance logs`.

---

As per our subsequent discussions with the team, we'll keep this logger.